### PR TITLE
added project settings table for godot3

### PR DESCRIPTION
**Pull request by ZX-WT**
I used the file mentioned in #395, I updated it to fit the latest project settings, I can't preview it on github so there should be some problem in the format, I will fix it soon.

### CODE REVIEW

**Comment by ZX-WT** [comment]
Sill cannot fix the malformed table error, can anyone tell me about the exact requirement for setting up the grid table?

File: learning/editor/godot_3_project_settings.rst [hunk 1]
```diff
@@ -0,0 +1,720 @@
+.. _doc_godot_3_project_settings:
+
+Godot 3 Project Settings
+========================
+
++---------------+-------------------------+
+|  Version | v3.0 beta2                   |
```
Comment (akien-mga) [inline learning/editor/godot_3_project_settings.rst]: You need to *draw* the table, everything should be properly aligned in RST. It's a PITA but that's how it was designed... So the `|` need to be below the `+` at all times.

File: learning/editor/godot_3_project_settings.rst [hunk 2]
```diff
@@ -0,0 +1,720 @@
...
+
+Depth Prepass
++++++++++++++
+
++----------------------------------------+---------------+
+| Enable                  | *Description* |
++----------------------------------------+---------------+
+| Disable For Vendors                  | *Description* |
++----------------------------------------+---------------+
+| Disable                  | *Description* |
```
Comment (akien-mga) [inline learning/editor/godot_3_project_settings.rst]: You can remove that option, it's a bug and will be removed.

**Comment by ZX-WT** [comment]
I am frustrated :( 
I have fixed the alignment in the file

But it looks different on github, is it only a font display problem?

**Comment by akien-mga** [comment]
Are you using tabs or a non-ascii space?

**Comment by pwab** [comment]
Not sure what you are using there but when I replaced all the whitespace chars with normal ascii space everything works fine again.

```
+---------------+-------------------------+
|  Version      | v3.0 beta2              |
+---------------+-------------------------+
|  Last Edit    | 07.01.2018              |
+---------------+-------------------------+
```

**Comment by mhilbrunner** [comment]
See my comment in https://github.com/godotengine/godot-docs/pull/965 about version & last edit.

**Comment by mhilbrunner** [comment]
Closing this for now, see comment in https://github.com/godotengine/godot-docs/pull/965.

**Pull request closed without merging**